### PR TITLE
Update gallery.blade.php

### PR DESCRIPTION
**Pull request by ag84ark**
On mobile view, the image was represented as more than 100% device width and the whole layout was broken. When you go to see the next image, the initial image gets the correct dimensions, but the mobile menu is still broken, you need to scroll to the right to be able to close it after you have opened. I am having this problem with a square image 1280x1280 that is displayed to 522x522.

### CODE REVIEW

**Review by shivendra-webkul — APPROVED** [review]

**Comment by shivendra-webkul** [comment]
Hi @ag84ark, There are conflicts in this branch. Please resolve them.

**Comment by ag84ark** [comment]
@shivendra-webkul Done

File: packages/Webkul/Shop/src/Resources/views/products/view/gallery.blade.php [hunk 1]
```diff
@@ -103,7 +103,7 @@ class="min-w-[450px] rounded-xl"
             </div>
 
             <!-- Product slider Image with shimmer -->
-            <div class="scrollbar-hide flex gap-8 overflow-auto 1180:hidden">
+            <div class="scrollbar-hide flex gap-8 overflow-auto 1180:hidden w-screen">
```
Comment (shivendra-webkul) [inline packages/Webkul/Shop/src/Resources/views/products/view/gallery.blade.php:106]: Please reorder the class add 'w-screen' after 'flex'

Comment (ag84ark) [reply packages/Webkul/Shop/src/Resources/views/products/view/gallery.blade.php]: @shivendra-webkul Done

**Review by shivendra-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**